topic-operator/src/main/java/io/strimzi/operator/topic/ConflictingChangesException.java

```diff
@@ -11,6 +11,12 @@
  * and the topic in Kafka had conflicting changes.
  */
 public class ConflictingChangesException extends OperatorException {
+
+    /** Constructor
```

Comment:
Eugh, can we put this on the following line? (And same for other instances, you might be able to fix this with a suitable `sed` invocation).

Reply:
Fixed @tombentley